checker/src/org/checkerframework/checker/genericeffects/qual/SafeCast.java

```diff
@@ -8,5 +8,8 @@
 
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
-@Target({ElementType.CONSTRUCTOR, ElementType.METHOD, ElementType.FIELD})
-public @interface NoIOEffect {}
```

Comment:
Again, please bring back the IOEffectChecker, including its qualifiers.  I'll help wrangle git in our Friday meeting if you like.